### PR TITLE
Fix Media Cache Handler

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

**Review by sebastienros — APPROVED** [review]
Interestingly this could have been done on purpose.

File: src/OrchardCore/OrchardCore.Media.Core/Events/DefaultMediaFileStoreCacheEventHandler.cs [hunk 1]
```diff
@@ -12,23 +12,11 @@ public DefaultMediaFileStoreCacheEventHandler(IMediaFileStoreCacheFileProvider m
     }
 
     public override Task MediaDeletedDirectoryAsync(MediaDeletedContext context)
-    {
-        _mediaFileStoreCacheFileProvider.TryDeleteDirectoryAsync(context.Path);
-
-        return Task.CompletedTask;
-    }
+        => _mediaFileStoreCacheFileProvider.TryDeleteDirectoryAsync(context.Path);
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Media.Core/Events/DefaultMediaFileStoreCacheEventHandler.cs:15]: I think we accepted the rule to use block body for methods, ctor and conversions. Only use expression body for properties and indexers.

Comment (MikeAlhayek) [reply src/OrchardCore/OrchardCore.Media.Core/Events/DefaultMediaFileStoreCacheEventHandler.cs:15]: I don't think there was an objection into using expression for one line like this.

**Comment by MikeAlhayek** [comment]
> Interestingly this could have been done on purpose.

Like fire and forget? If so, that would be done at the higher level not here. I don't think there is a reason to execute a task and return a completed task. Maybe I am missing something here

File: src/OrchardCore/OrchardCore.Media.Core/Events/DefaultMediaFileStoreCacheEventHandler.cs (hunk 1 above)
Comment (sebastienros) [reply src/OrchardCore/OrchardCore.Media.Core/Events/DefaultMediaFileStoreCacheEventHandler.cs:15]: I looked at the editor config and watched the meeting. You are correct, for methods we "can" use expression bodies when it is on a single line.

**Pull request merged**